flight/PiOS/Common/pios_mpu.c

```diff
@@ -216,8 +217,6 @@ static int32_t PIOS_MPU_Validate(struct pios_mpu_dev *dev)
 		return -1;
 	if (dev->magic != PIOS_MPU_DEV_MAGIC)
 		return -2;
-	if (dev->com_driver_id == 0)
-		return -3;
 	return 0;
```

Comment:
Fair enough. The only thing it can really protect is if someone steps over the dev struct with 0s, we'd not try to access it, but that's very much an edge-case.